src/App.jsx

```diff
@@ -0,0 +1,13 @@
+import './App.css';
+import HomePage from './components/home';
+
+function App() {
+  return (
+    <>
+      <HomePage />
+    </>
```

Comment:
- [ ] [OPTIONAL] You have done great work @Daniel-Kigozi 💪. However, I'll suggest you ameliorate your design : 

- To make your buttons more beautiful, kindly you may just add some styles 👍 

![Screenshot 2023-06-01 184035](https://github.com/Daniel-Kigozi/react-capstone-air-pollution/assets/57408419/39343873-0b0f-4e54-bdcb-5a5cca132c1c)

- Also add some `margin` and `padding ` on your country section 👍 

![Screenshot 2023-06-01 184105](https://github.com/Daniel-Kigozi/react-capstone-air-pollution/assets/57408419/f04bd6fd-83a5-4f63-a536-5969c4e8a483)